docs/source/nodeoperations/restore.md

```diff
@@ -2,88 +2,223 @@
+* `CLUSTER_ID` - is a name or ID of registered cluster having access to `BACKUP_LOCATION`. 
+* `BACKUP_LOCATION` - is a location where backup is stored.
 
-```bash
-sctool backup list -c simple-cluster --all-clusters -L s3:backups
+In our example `BACKUP_LOCATION` is `s3:source-backup` and we will use `scylla/target` cluster as `CLUSTER_ID` to make sure it has access to bucket.
+
+```console
+$ sctool backup list -c scylla/target --all-clusters -L s3:source-backup
+backup/ff36d7e0-af2e-458c-afe6-868e0f3396b2
 Snapshots:
-  - sm_20201227144037UTC (409MiB)
-  - sm_20201228145917UTC (434MiB)
+  - sm_20240105115931UTC (409MiB, 1 nodes)
 Keyspaces:
+  - system_schema (15 tables)
   - users (9 tables)
-  - system_auth (2 tables)
-  - system_distributed (3 tables)
-  - system_schema (13 tables)
-  - system_traces (5 tables)
+
 ```
 
-To get the list of files use:
+We are going to restore the cluster using `sm_20240105115931UTC` snapshot.
 
-```bash
-sctool backup files -c <CLUSTER_ID> -L <BACKUP_LOCATION> -T <SNAPSHOT_TAG>
+Restoring consist of two steps, first we restore the schema, and then the data.
+
+To restore schema, create a restore task manually on target ScyllaCluster by executing following command in Scylla Manager Pod.
+```console
+sctool restore -c <CLUSTER_ID> -L <BACKUP_LOCATION> -T <SNAPSHOT_TAG> --restore-schema
 ```
 
 Where:
-* `SNAPSHOT_TAG` - name of snapshot you want to restore.
-
-Before we start restoring the data, we have to restore the schema.
-The first output line is a path to schemas archive, for example:
-```bash
-s3://backups/backup/schema/cluster/ed63b474-2c05-4f4f-b084-94541dd86e7a/task_287791d9-c257-4850-aef5-7537d6e69d90_tag_sm_20201228145917UTC_schema.tar.gz      ./
+* `CLUSTER_ID` - is a name or ID of a cluster we are restoring to.
+* `BACKUP_LOCATION` - location where backup is stored.
+* `SNAPSHOT_TAG` - snapshot tag of a backup we are restoring from.
+
+After task is created, output of the command will return ID of a restore task.
+```console
+$ sctool restore -c scylla/target -L s3:source-backup -T sm_20240105115931UTC --restore-schema
+restore/57228c52-7cf6-4271-8c8d-d446ff160747
 ```
 
-To download this archive you can use AWS CLI tool `aws s3 cp`.
-
-This archive contains a single CQL file for each keyspace in the backup.
-```bash
-tar -ztvf task_287791d9-c257-4850-aef5-7537d6e69d90_tag_sm_20201228145917UTC_schema.tar.gz
--rw------- 0/0           12671 2020-12-28 13:17 users.cql
--rw------- 0/0            2216 2020-12-28 13:17 system_auth.cql
--rw------- 0/0             921 2020-12-28 13:17 system_distributed.cql
--rw------- 0/0           12567 2020-12-28 13:17 system_schema.cql
--rw------- 0/0            4113 2020-12-28 13:17 system_traces.cql
+To check progress use:
+```console
+$ sctool progress -c scylla/target restore/57228c52-7cf6-4271-8c8d-d446ff160747
+Restore progress
```

Comment:
can we hide the outputs? Iirc the plugin we're using allows to display it on-click and hide by default.

Reply:
I tried, with admonition having toggle class but it didn't work. It looked bad, and toggle was missing. Haven't spend more time on it so leaving as is.